### PR TITLE
Updating readme with conditions for adding image and fixing headings.

### PR DESCRIPTION
**Pull request by mohammedzee1000**
The readme should explicitly mention the conditions for adding image
to index. Also, correcting the levels of the headings.

Signed-off-by: Mohammed Zeeshan Ahmed <mohammed.zee1000@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,16 @@
 
 This is the master index used by the [CentOS Community Container Pipeline Service](https://github.com/CentOS/container-pipeline-service) to build and deliver containers at [registry.centos.org](https://wiki.centos.org/ContainerPipeline)
 
-# Quickstart
+## Adding your project to index
+
+Any project is welcome to add their containers to the container index, so long as
+
+1. Project is opensource, and they are ok with their image being published publically on registry.centos.org, alongwith the logs, such as build logs, scan logs etc.
+2. Image is CentOS based (eg: dockerfile has ```FROM centos``` or its eventual base image).
```
Comment (dharmit) [inline README.md]: Suggestion:
Image is CentOS based, i.e., Dockerfile has ```FROM centos``` or its eventual base image.

**Comment by mohammedzee1000** [comment]
@dharmit thanks, updated

File: README.md [hunk 2]
```diff
@@ -2,7 +2,16 @@
 
 This is the master index used by the [CentOS Community Container Pipeline Service](https://github.com/CentOS/container-pipeline-service) to build and deliver containers at [registry.centos.org](https://wiki.centos.org/ContainerPipeline)
 
-# Quickstart
+## Adding your project to index
+
+Any project is welcome to add their containers to the container index, so long as
+
+1. Project is opensource, and they are ok with their image being published publically on registry.centos.org, alongwith the logs, such as build logs, scan logs etc.
```
Comment (navidshaikh) [inline README.md]: *along with the long

missing whitespace between `along` `with`

Comment (navidshaikh) [reply README.md]: *Open-Source or Open Source

File: README.md [hunk 3]
```diff
@@ -2,7 +2,16 @@
...
+2. Image is CentOS based i.e Dockerfile has ```FROM centos``` or its eventual base image does).
```
Comment (navidshaikh) [inline README.md]: missing matching starting `(`

**Comment by mohammedzee1000** [comment]
@navidshaikh thanks, fixing

**Pull request merged**